### ISSUE TITLE
Sunricher HK-ZCC-ZLL-A: window covering device

### ISSUE THREAD

**Issue by kvanisterdael** [dresden-elektronik/deconz-rest-plugin#8041]
### Is there already an existing issue for this?

- [X] I have searched the existing issues and there is none for my device

### Product name

Zigbee Curtain Motor Controller SR-ZG9080A

### Manufacturer

Sunricher

### Model identifier

HK-ZCC-ZLL-A

### Device type to add

Other

### Node info

![afbeelding](https://github.com/user-attachments/assets/fe4cc234-7562-4716-8e27-02972101f318)

### Endpoints and clusters

![afbeelding](https://github.com/user-attachments/assets/c1325816-46b6-4603-8921-658858776dde)

### Basic

![afbeelding](https://github.com/user-attachments/assets/d6dd47b3-b542-4339-8af1-fb2265ea2aca)

**Comment by kvanisterdael** [comment]
Device is exactly the same as the one defined in SR-ZG9080A.json.

**Comment by kvanisterdael** [comment]
Proposal for DDF file: 
[hk-zcc-zll-a.json](https://github.com/user-attachments/files/17904563/hk-zcc-zll-a.json)

**Comment by Smanar** [comment]
Hello, but the support is not native for this device ?

**Comment by kvanisterdael** [comment]
No, in v 2.28.1 it didn't work out of the box like the previous device (SR-ZG9080A.json) did.
